modules/local/pave/germline/main.nf

```diff
@@ -0,0 +1,82 @@
+// NOTE(SW): use of tumor sample name here is consistent with Pipeline5
+//  - https://github.com/hartwigmedical/pipeline5/blob/v5.33/cluster/src/main/java/com/hartwig/pipeline/tertiary/pave/PaveGermline.java#L36-L41
+//  - https://github.com/hartwigmedical/pipeline5/blob/v5.33/cluster/src/main/java/com/hartwig/pipeline/tertiary/pave/PaveArguments.java#L31-L43
+
+import nextflow.Nextflow
```

Comment:
What is this for?

Reply:
The import is so that `Nextflow.exit(1)` can be called on an bad genome version being provided. Similar to the assert False you've referenced in main.nf, this is done out of an abundance of caution and with the current set up it should never execute unless something has gone severely wrong. It serves more as a safeguard for developers when making changes.

Since this is a local module I think this is okay - did you have any recommendations?

Reply:
This is much better, thank you!

<br />

- Change made in https://github.com/nf-core/oncoanalyser/pull/68